Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.cpp

```diff
@@ -103,12 +103,16 @@ namespace ROS2::Utils
 
     UrdfAssetMap FindAssetsForUrdf(const AZStd::unordered_set<AZStd::string>& meshesFilenames, const AZStd::string& urdFilename)
     {
+        auto enviromentalVariable = std::getenv("AMENT_PREFIX_PATH");
```

Comment:
question: Do we have an `AZStd` equivalent of this? 🤔 May well be fine but it might be useful to add an alias in future.

e.g.
```
namespace AZStd
{
    using std::env;
}
```

In a more central location (we might be able to follow-up with @o3de/sig-core on this)

Reply:
I am leaving here std::env for now.